crypto.pl

```diff
@@ -33,7 +33,12 @@
 */
 
 :- module(crypto,
-          [ evp_decrypt/6,              % +CipherText, +Algorithm, +Key, +IV, -PlainText, +Options
+          [ crypto_term_hash/3,         % +Data, -Hash, +Options
```

Comment:
Personally, I would still prefer `crypto_term_hash/3`, with the prospect to extend this in the future to more terms. I originally chose this term exactly because `term_hash/2` is already known. However, `crypto_data_hash/3` is also sufficiently general to allow&nbsp;this, and matches the variable names that are used in the documentation of the existing hash predicates to refer to the&nbsp;data.